core/modules/layout/tests/layout.test

```diff
@@ -1190,3 +1190,179 @@ class LayoutUpgradePathTest extends UpgradePathTestCase {
     return $this->assertBlocks($blocks, TRUE);
   }
 }
+
+/**
+ * Tests invocation of hooks when performing an action.
+ *
+ * Tested hooks are:
+ * - hook_layout_presave
+ * - hook_layout_insert
+ * - hook_layout_update
+ * - hook_layout_disable
+ * - hook_layout_enable
+ * - hook_layout_delete
+ * - hook_layout_block_remove
+ * - hook_layout_template_change
+ * - hook_layout_revert
+ */
+class LayoutHookTestCase extends BackdropWebTestCase {
+  protected $admin_user;
+
```

Comment:
If it's possible, it's good to use a lighter-weight profile than the standard profile (avoiding unneeded modules), it makes for a faster test.

For this test, I'd recommend using the 'testing' profile with:

```
protected $profile = 'testing';
```